src/components/media/Media.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+// the outermost element
+.bdl-Media {
+    align-items: flex-start;
+    display: flex;
+}
+
+// the media content, ie, an avatar
+.bdl-Media-figure {
+    align-self: flex-start;
+    flex: 0 0 auto;
+    justify-self: flex-start;
+    margin: 0;
+    padding: 0;
+}
+
+// the main content, e.g. title and message
+.bdl-Media-body {
+    flex: 1 1 auto;
+    margin-left: 10px;
+    // handle wrapping long words in text content without breaking
+    // float position of menu
+    word-break: break-word;
+}
+
+// the "..." menu button, meant to be used inside Media-body
+// float is used to reflow content instead of pushing it to the left
+.bdl-Media-menu {
+    float: right;
+}
+
+// overrides for .btn-plain's margin resets on focus states
+.bdl-Media-menu,
```

Comment:
Minor, but this can be modified to avoid the repetition of the class name:

```
.bdl-Media-menu {
    &,
    &:active,
    &:hover,
    &:focus {
        margin-bottom: 5px;
        margin-left: 5px;
    }
}
```